### ISSUE TITLE
Accessibility: Back arrow for navigation misplaced and position should be customizable

### ISSUE THREAD

**Issue by bugith** [NothingOSS/android_kernel_msm-5.4_nothing_sm7325#5]
For right-handed/single arm people using System Navigation Gestures with the so big screen. Hitting the back arrow is a pain. It should be nice if we could adjust the position in the same way we already have a setting (right/left) for classical Three-button Navigation, more the possibility to set it on top or bottom of the screen. 
![Screenshot_20230104-230358](https://user-images.githubusercontent.com/6240207/210660836-021ac8d7-7728-41d6-a2f0-5cf48e598563.png)
